charts/vector/values.yaml

```diff
@@ -365,6 +365,21 @@ persistence:
   # "Aggregator" role.
   # storageClassName: default
 
+  # The [Persistent Volume Claim (PVC) retention policy](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#persistentvolumeclaim-retention)
+  # controls if and how PVCs are deleted during the lifecycle of a StatefulSet.
+  # WhenDeleted specifies what happens to PVCs created from StatefulSet VolumeClaimTemplates when the StatefulSet is deleted, 
+  # and WhenScaled specifies what happens to PVCs created from StatefulSet VolumeClaimTemplates when the StatefulSet is scaled down.
+  # 
+  # Example:
+  #
+  # ```yaml
+  # persistentVolumeClaimRetentionPolicy:
+  #   whenDeleted: Retain
+  #   whenScaled: Retain
+  # ```
+  # @type: map
+  persistentVolumeClaimRetentionPolicy: {}
```

Comment:
`persistence.persistentVolumeClaimRetentionPolicy` is quite lengthy and redundant. What do you think about naming this `persistence.retentionPolicy` instead?

Reply:
Sure, will do it! The reason of using the whole name is because this is the name of the variable in Kubernetes, but it might be redundant as you said 👍

Reply:
Hello again! I've executed the requested actions, but according to the README, I think they should be executed on `develop` after merging into `main`. 

- Chart is using the latest Vector version, so I didn't bumped it.
- Updated the Chart version in Chart.yaml from `0.29.0` to `0.30.0`.
- Updated Changelog manually: I've created a new commit (otherwise I can't reference the commit itself in the Changelog)
- I've renamed the variable name as proposed
- I've updated the tests in the description (and I've run them again to validate that it works fine)
- I've updated the commit to include the PR and to be a conventional commit ( `enhance(): ...`)

Hope it's fine now! Please let me know if I should run any other action 🙏 

Many thanks 😄